lib/projects/pc-fish.js

```diff
@@ -7,9 +7,9 @@
   "phase":"beta",
   "phase_modifier":"development",
```

Comment:
We talked about moving this to Beta Public, in which this needs to be changed from `development` to `public`